pkg/gui.py

```
@@ -190,152 +213,132 @@ def check_status(self, thread):
+    def display_body_message(self, message: str, display_time: int = None) -> None:
+        '''
+        displays a message in the main body of the app
+        '''
+        if display_time is None:
+            display_time = self.alive_time
+        label = Label(self.body_frame, text=message)
+        label.grid(row=0, column=0, pady=2)
+        self.body_frame.after(display_time-100, lambda: label.destroy())
+        self.body_frame.update()
+
+    def add_to_history(self, message_item) -> None:
```

Ruff (0.8.0)

Missing type annotation for function argument `message_item`

(ANN001)